code/modules/clothing/gloves/color.dm

```diff
@@ -45,6 +46,9 @@
 
 /obj/item/clothing/gloves/color/black/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
 	if(istype(W, /obj/item/weapon/wirecutters))
+		if(!can_be_cut)
+			..()

```

Comment:
the `..()` isn't needed. If you really want the parent to be called in all cases, just do if(can_be_cut) stuff stuff